shipitsteven/note-app
# Ss/db interface

## Description of Issue
Implementing the file store saving system for notes. Saves notes into a folder called notes which is created within the client folder of the project.

## Testing
Also created a tests file called note_storage.test.ts which uses yarn test to run the tests. Tests use jest library for testing.

### To Test
Check the code to see if everything looks good, there were also some changes made to some config files because of using jest for testing.

## Time Spent
- *How long did you estimate this task would take?* 6 hours
- *How long did it take you to work on this issue?* 8 hours

## Checklist
